tracer/src/Datadog.Trace/Configuration/IntegrationId.cs

```diff
@@ -41,5 +41,6 @@ internal enum IntegrationId
         Oracle,
         SqlClient, // SQL Server
         Sqlite,
+        AdoNet,
```

Comment:
(I brought this up elsewhere, repeating it here for the record.)

We should discuss how configuration will work with the return of `AdoNet` after the recent changes in #2008 and whether we want to revert those breaking changes for tracer 2.0.